commands/usernote.js

```diff
@@ -95,9 +95,28 @@ module.exports = {
           }
         });
         if (notes) {
-          return Promise.map(notes.ns, note => formatNote(note, args.subreddit, channel))
-            .map((note, index) => `(${index}) ${note}`)
-            .then(formattedNotes => [`Notes on /u/${args.user} on /r/${args.subreddit}:`].concat(formattedNotes));
+          if(!(args.all === false)) {
+              return Promise.map(notes.ns, note => formatNote(note, args.subreddit, channel))
+                .map((note, index) => `(${index}) ${note}`)
+                .then(formattedNotes => [`Notes on /u/${args.user} on /r/${args.subreddit}:`].concat(formattedNotes));
+          } else {
```

Comment:
Instead of duplicating the code to format the notes, can you use the same code in both cases?

```js
return Promise.map(
    args.all ? notes.ns : notes.ns.slice(0, 5),
    note => formatNote(note, args.subreddit, channel)
)
  .map((note, index) => `(${index}) ${note}`)
  .then(formattedNotes => [`Notes on /u/${args.user} on /r/${args.subreddit}:`].concat(formattedNotes))
```